### ISSUE TITLE
<tbody> not being found in a WELL FORMED html code

### ISSUE THREAD

**Issue by azev** [cheeriojs/cheerio#1007]
Cheerio is failing to get rows (`<tr>`) when parsing a **well formed** html.

```
<table id="mytable">
  <thead>
    <tr>
      <th>Month</th>
      <th>Savings</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>January</td>
      <td>$100</td>
    </tr>
    <tr>
      <td>February</td>
      <td>$80</td>
    </tr>
  </tbody>
  <tfoot>
    <tr>
      <td>Sum</td>
      <td>$180</td>
    </tr>
  </tfoot>
</table>
```

`$('#mytable > tbody > tr')` will return empty.

I have to do:

`$('#mytable tr').slice(1,-1)` to remove header and footer rows, since it will get `<thead>` and `<tfoot>`.

**Comment by DianeLooney** [comment]
It seems to work for me. [Link](https://gist.github.com/DianeLooney/f2f2a8888440d98decb9826acc429701)
What version of cheerio are you using, and what version of node?

**Comment by Vladislao** [comment]
I think element *\<tbody\>* is missing in the initial markup, but automatically included in the DOM when you try to browse code. You can read more about it [here](http://stackoverflow.com/questions/938083/why-do-browsers-insert-tbody-element-into-table-elements)

**Comment by azev** [comment]
I see. Thanks for the answers guys. I'll look into the actual HTML when I get my hands on the code again. Probably when I click "View page source" on Firefox it automatically insert the `tbody` as you mentioned.

**Comment by fb55** [comment]
We are currently working on replacing the HTML parser, so this will be fixed with #985

**Comment by azev** [comment]
Thanks everyone.

**Comment by fb55** [comment]
Fixed by #985
